lib/Command/Lock.php

```diff
@@ -136,14 +136,16 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			$this->getStatus($input, $output, $fileId);
 			$this->unlockFile($input, $output, $fileId);
 		} catch (SuccessException $e) {
-			return;
+			return 0;
```

Comment:
We could set the return type for this method. Or does it break the signature and then Php complains about a superclass method not being overloaded?